### ISSUE TITLE
Pass custom function to render the interface of column filtering

### ISSUE THREAD

**Issue by nuno-agostinho** [rstudio/DT#349]
I dislike how the column filtering works when the user desires to input small numbers (such is the case with p-values). Although the user can manually edit the values, they are hard to edit if they have many decimal places because of the input width.

This would be easy to solve if I had access to the `filterRow` function used. In my case, I'd prefer to manually edit the values in the dropdown that appears (click on the min or max value and allow the user to edit it in-place).

In short, using a custom function (as an argument passed to `datatable`) instead of `filterRow` would be great!
